.gitignore

```diff
@@ -7,4 +7,8 @@ extensions/test-src/solution/test/tmp/
 build-utils/dist
 build-utils/lib
 build-utils/test-lib
-.settings
\ No newline at end of file
+.settings
+.DS_Store
+override.properties
+build-res/
```

Comment:
Not sure about this one (build-res). we don't want git to ignore it, jsut developers not to change it. Adding it here will mess up things for the automated scripts that update these files automatically.